app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,244 @@
  *= require_tree .
  *= require_self
  */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  transition: all 0.3s ease-in-out;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+body {
+  background-color: #2b2b2b;
+}
+
+main {
+  max-width: 500px;
+  margin: 0 auto;
+  padding-bottom: 2rem;
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  color: #434b54;
+  min-height: 740px;
+  margin-top: 2vh;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+#notice,
+#alert {
+  text-align: center;
+  padding: 0.5rem;
+  background-color: #3778c2;
+  color: #fff;
+}
+
+.header {
+  background-color: #3778c2;
+  color: #fff;
+  padding: 1rem 0.6rem;
+}
+
+.header h1 {
+  font-size: 1.4rem;
+}
+
+#splash-screen {
+  width: 100%;
+  height: 80vh;
+  display: grid;
+  grid-template-rows: 3fr 1fr;
+}
+
+#splash-screen h1 {
+  display: grid;
+  place-items: center;
+}
```

Comment:
- [ ] Consider using reusable classes for reoccurring CSS rules. For example, `display: grid;` is used several times throughout this file, Kindly create a reusable class and add that class name to the affected elements